src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/InterventionServiceTest.kt

```diff
@@ -85,6 +87,23 @@ class InterventionServiceTest @Autowired constructor(
 //    assertThat(interventionDTO.pccRegions.size).isEqualTo(1)
   }
 
+  @Test
```

Comment:
Should testing retrieving data from the db be done as a repository test. I think elsewhere service tests have mocked out the repositories?

Although, I can see that other tests in this file are doing it the way you have below

Reply:
i'm really keen for this sort of test to actually involve the database, especially since we want to test that the db itself does not modify the character encoding or formatting.